server/src/main/resources/registry.conf

```diff
@@ -1,6 +1,6 @@
 registry {
-  # file 、nacos 、eureka、redis、zk
-  type = "file"
+  # file 、nacos 、eureka、redis、zk、consul
+  type = "consul"
```

Comment:
default type= file

Reply:
i will change it back.it was changed for testing.